server/services/FiltersService/FiltersService.test.js

```diff
@@ -28,7 +28,20 @@ describe(FiltersService, () => {
 
         expect(service.filters).toEqual({
           ...initialFilters,
-          sites: {},
+          sites: {
+            one: {
+              label: 'one',
+              value: 0,
```

Comment:
should these values be different, vs all `0`? Ohhh wait, 0 means not selected, right?

Reply:
Yup!